blog/using-duckdb-with-wrds/index.qmd

```diff
@@ -0,0 +1,301 @@
+---
+title: "Using DuckDB with WRDS Data"
+author: "Ian Gow"
+date: "2023-06-08"
+description: Demonstrate the power of DuckDB and dbplyr with WRDS data.
+image: thumbnail.png
+image-alt: image alt text
+categories: 
+  - Data
+---
+
+In this short note, I show how one can use DuckDB with WRDS data stored in the PostgreSQL database provided by WRDS.
```

Comment:
I also believe that a few sentences about the idea behind DuckDB and why it makes sense to use it in academic contexts as well would be great here.